src/SmartComponents/Portfolio/remove-portfolio-modal.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Modal, Button } from '@patternfly/react-core';
+import { addNotification } from '@red-hat-insights/insights-frontend-components/components/Notifications';
+import { removePortfolio } from '../../redux/Actions/PortfolioActions';
+import { hideModal } from '../../redux/Actions/MainModalActions';
+import { pipe } from 'rxjs';
+
+const RemovePortfolioModal = ({
+  history: { goBack, push },
+  removePortfolio,
+  addNotification,
+  hideModal,
+  portfolioId,
+  portfolioName
+}) => {
+  const onSubmit = () => removePortfolio(portfolioId).then(hideModal()).then(push('/portfolios'));
```

Comment:
Missing refetch of portfolios, deleted portfolio will remain in redux until page reload.